### DISCUSSION TITLE
Dnd: Maintaining parent isDropTarget state when hovering over child drop target

### DISCUSSION THREAD

**Discussion by vrimar** [adobe/react-spectrum#6944] [Q&A]
Is there a way to maintain the parent component's `isDropTarget` truthy state when hovering over a child drop target as seen in this example:

[Codesandbox](https://codesandbox.io/p/sandbox/nifty-heisenberg-ntl7ds?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm09lz4is0006356nwqkoldv0%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm09lz4is0002356n6mp0cwtq%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm09lz4is0003356nmeh1no6n%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm09lz4is0005356n1k7lpc1s%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm09lz4is0002356n6mp0cwtq%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm09lz4ir0001356n8bk49l37%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.tsx%2522%257D%255D%252C%2522id%2522%253A%2522cm09lz4is0002356n6mp0cwtq%2522%252C%2522activeTabId%2522%253A%2522cm09lz4ir0001356n8bk49l37%2522%257D%252C%2522cm09lz4is0005356n1k7lpc1s%2522%253A%257B%2522id%2522%253A%2522cm09lz4is0005356n1k7lpc1s%2522%252C%2522activeTabId%2522%253A%2522cm09lz5my000p356nja908krg%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm09lz4is0004356nyvd37u86%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522id%2522%253A%2522cm09lz5my000p356nja908krg%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522cm09lz4is0003356nmeh1no6n%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522cm09lz4is0003356nmeh1no6n%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

Notice how ParentDrop's `isDropTarget` is set to false when hovering over a ChildDrop component.

**Comment by snowystinger** [comment]
You can only drop on one thing at a time. Can you make the children always visible? you could also put an onDropEnter/Exit for each of the children and persist in showing the children when that is called?

**Comment by vrimar** [comment]
Apologies for the lack of context, I'm trying to re-create this existing flow I currently have with react-dnd:

![image](https://github.com/user-attachments/assets/d1141a32-144d-41e9-880d-4d1cb6fe199e)

React-dnd maintains it's `isDropTarget` state when hovering over child items within a parent container, is there perhaps a better approach for this in react-aria?

**Reply by snowystinger** [reply]
Were you able to try
> you could also put an onDropEnter/Exit for each of the children and persist in showing the children when that is called?
